### PR TITLE
DEVX-2676: Updating commands to match new CLI

### PR DESCRIPTION
**Pull request by bbejeck**
### Description

Fixes issue https://confluentinc.atlassian.net/browse/DEVX-2676

Validated against the following tutorials 

- [Console Producer and Consumer with Deserializers](http://kafka-tutorials-staging.s3-website-us-west-.amazonaws.com/DEVX-2676_update_for_new_cli/kafka-console-consumer-producer-avro/confluent.html)
- [Join a stream and lookup table](http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/DEVX-2676_update_for_new_cli/join-a-stream-to-a-table/confluent.html)
- https://github.com/confluentinc/kafka-tutorials/pull/983 uses the new CLI as well

### Staging Docs
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/DEVX-2676_update_for_new_cli/

### CODE REVIEW

File: _includes/tutorials/sliding-windows/kstreams/markup/dev/generate-data.adoc.bak [hunk 1]
```diff
@@ -1,44 +0,0 @@
...
-Date: Tue, 26 Jan 2021 21:40:33 GMT
-Location: http://localhost:8083/connectors/datagen_local_01
-Content-Type: application/json
-Content-Length: 413
-Server: Jetty(9.4.24.v20191120)
-
-{"name":"datagen_local_01","config":{"connector.class":"io.confluent.kafka.connect.datagen.DatagenConnector","key.converter":"org.apache.kafka.connect.storage.StringConverter","kafka.topic":"temp-readings","schema.filename":"/schemas/datagen-temperature-reading.avsc","schema.keyfield":"device_id","max.interval":"300","iterations":"10000000","tasks.max":"1","name":"datagen_local_01"},"tasks":[],"type":"source"}
-----
-
-
```
Comment (bbejeck) [inline _includes/tutorials/sliding-windows/kstreams/markup/dev/generate-data.adoc.bak:44]: This file appears to have been committed by accident before, so deleting this as cleanup on the side

File: _includes/tutorials/sliding-windows/kstreams/markup/dev/run-consumer.adoc.bak [hunk 2]
```diff
@@ -1,30 +0,0 @@
...
-device-1 : 82.06
-device-2 : 73.63
-device-2 : 81.75
-device-1 : 86.32
-device-1 : 82.82
-</code></pre>
-++++
-
-
-Go ahead and shut down your streams application now with a `CNTR+C` command.
```
Comment (bbejeck) [inline _includes/tutorials/sliding-windows/kstreams/markup/dev/run-consumer.adoc.bak:30]: This file appears to have been committed by accident before, so deleting this as clean up on the side

File: _includes/shared/markup/ccloud/get-ccloud.adoc [hunk 3]
```diff
@@ -1,2 +1,2 @@
 This tutorial has some steps for Kafka topic management and/or reading from or writing to Kafka topics, for which you can use the Confluent Cloud Console or install the Confluent Cloud CLI.
-Instructions for installing Confluent Cloud CLI and configuring it to your Confluent Cloud environment is available from within the Confluent Cloud Console: navigate to your Kafka cluster, click on the `CLI and tools` link, and run through the steps in the `CCloud CLI` tab.
+Instructions for installing Confluent Cloud CLI and configuring it to your Confluent Cloud environment is available from within the Confluent Cloud Console: navigate to your Kafka cluster, click on `Dashboard` from the list on the left, then click the `Set up the CLI` link, and run through the steps in the `Set up CLI` page.
```
Comment (ybyzek) [inline _includes/shared/markup/ccloud/get-ccloud.adoc:2]: 1. I don't see what you see (Dashboard, Set up CLI, etc) ;) 
2. I think the CLI has been renamed from `Confluent Cloud CLI` to just `Confluent CLI` -- please confirm, but so, then the text should change here and throughout all KT headings (in the harness files)

**Review by ybyzek — APPROVED** [review]
@bbejeck once you add a link to https://docs.confluent.io/confluent-cli/current/migrate.html, LGTM and thank you!

File: _includes/shared/markup/ccloud/get-ccloud.adoc [hunk 4]
```diff
@@ -1,2 +1,4 @@
...
-Instructions for installing Confluent Cloud CLI and configuring it to your Confluent Cloud environment is available from within the Confluent Cloud Console: navigate to your Kafka cluster, click on the `CLI and tools` link, and run through the steps in the `CCloud CLI` tab.
+This tutorial has some steps for Kafka topic management and/or reading from or writing to Kafka topics, for which you can use the Confluent Cloud Console or install the Confluent CLI.
+Instructions for installing Confluent CLI and configuring it to your Confluent Cloud environment is available from within the Confluent Cloud Console: navigate to your Kafka cluster, click on the `CLI and tools` link, and run through the steps in the `Confluent CLI` tab.
+
+NOTE: The CLI clients Confluent Cloud (`ccloud`) and Confluent Platform (`confluent`) have been unifed into a single client Confluent CLI (confluent v2.0).  The `ccloud` client will continue to work until May 9, 2022, but will be sunset after that point.  You can read the migration instructions and details for moving to `confluent` at https://docs.confluent.io/confluent-cli/current/migrate.html .
```
Comment (ybyzek) [inline _includes/shared/markup/ccloud/get-ccloud.adoc:4]:
```suggestion
NOTE: The CLI clients for Confluent Cloud (`ccloud`) and Confluent Platform (`confluent` v1.0) have been unified into a single client Confluent CLI `confluent` v2.0.  This tutorial uses the unified Confluent CLI `confluent` v2.0 (`ccloud` client will continue to work until sunset on May 9, 2022, and you can read the migration instructions to the unified `confluent` CLI in the https://docs.confluent.io/confluent-cli/current/migrate.html[documentation]).
```

**Pull request merged**